woocommerce/woocommerce-gutenberg-products-block
# Add 'Order notes'  to Checkout block

Fixes #1483.

### Screenshots

![imatge](https://user-images.githubusercontent.com/3616980/87674576-76b71600-c776-11ea-899e-70850108fde3.png)

### How to test the changes in this Pull Request:

1. Go to the Checkout block and check the 'Add a note to your order' checkbox.
2. Introduce some text in the text area that appeared.
3. Place the order and go to the admin, WooCommerce > Orders and verify the order has the note under `Customer provided note`.

### Changelog

> The Checkout block allows customers to introduce an order note. This feature can be disabled in the editor.
